content/projects/tilde/basic-repo-card-tutorial/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+content_type: project
+flavours:
+- none
+prerequisites:
+  hard:
+  - topics/github/helloworld
+  - environment-setup
+ready: true
+submission_type: repo
+title: 'Tilde project tutorial: Simple repository projects'
+protect_main_branch: false
+---
+
+There are a few kinds of projects you can do and they can be submitted in different ways. In this tutorial, you'll see how to get a repository project to complete.
```

Comment:
Initially people wont need to know about PRs or fancy stuff to do with the Review column. They just commit to main